common/djangoapps/student/views.py

```diff
@@ -46,6 +46,8 @@
 
 from edxmako.shortcuts import render_to_response, render_to_string
 
+from enterprise.tpa_pipeline import active_provider_requests_data_sharing, active_provider_enforces_data_sharing
+from enterprise.models import EnterpriseCustomer
```

Comment:
I would prefer that all references to enterprise app modules/workflows be routed through a "helpers" module -- see [organizations_helpers.py](https://github.com/edx/edx-platform/blob/master/common/djangoapps/util/organizations_helpers.py) for an example of this.